modules/xphoto/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 set(the_description "Addon to basic photo module")
-ocv_define_module(xphoto opencv_core opencv_imgproc WRAP python)
+ocv_define_module(xphoto opencv_core opencv_imgproc opencv_highgui WRAP python)
```

Comment:
Why do you need `higui` here? We are trying to reduce dependencies for each module.